File: Android/src/uk/co/kgutteridge/rnibhack/EPGModel/Channel.java
package uk.co.kgutteridge.rnibhack.EPGModel;

import com.google.gson.annotations.SerializedName;

public class Channel
{
	@SerializedName("channelno")
	public String channelNo;
	
	@SerializedName("epggenre")
	public String epggenre;
	
	@SerializedName("title")
	public String title;
	
	@SerializedName("channeltype")
	public String channelType;
	
	@SerializedName("channelid")
	public String channelID;
	
	@SerializedName("genre")
	public String genre;
}

File: iPhone/SkyEPG/SkyEPG/Channel.m
//
//  Channel.m
//  SkyEPG
//
//  Created by Kieran Gutteridge on 12/02/2012.
//  Copyright (c) 2012 __MyCompanyName__. All rights reserved.
//

#import "Channel.h"

@implementation Channel

@synthesize channelNo, epgGenre, title, channelType, channelID, genre;

-(void)dealloc
{
    self.channelNo = nil;
    self.epgGenre = nil;
    self.title = nil;
    self.channelType = nil;
    self.channelID = nil;
    self.genre =nil;
    
    [super dealloc];
}


@end
